tor_archivist/main.py

```diff
@@ -13,7 +13,16 @@
 from tor_archivist import __version__
 
 
+thirty_minutes = 1800  # seconds
+
+
 def run(config):
+    if (config.last_run + thirty_minutes) <= time.time():
```

Comment:
Not of vital importance, but I think it would make more sense to calculate `last_run + thirty_minutes` beforehand (and call it `sleep_until` or something like that) which I think would make it clearer what's happening here.

Reply:
Good point. It’s one less calculation to do on the fly. Will change.